Generals/Code/Libraries/Source/WWVegas/WWLib/RAWFILE.H

```diff
@@ -178,28 +178,4 @@ class RawFileClass : public FileClass
 		bool Allocated;
 };
 
-
-/***********************************************************************************************
- * RawFileClass::File_Name -- Returns with the filename associate with the file object.        *
- *                                                                                             *
- *    Use this routine to determine what filename is associated with this file object. If no   *
- *    filename has yet been assigned, then this routing will return NULL.                      *
- *                                                                                             *
- * INPUT:   none                                                                               *
- *                                                                                             *
- * OUTPUT:  Returns with a pointer to the file name associated with this file object or NULL   *
- *          if one doesn't exist.                                                              *
- *                                                                                             *
- * WARNINGS:   none                                                                            *
- *                                                                                             *
- * HISTORY:                                                                                    *
- *   10/18/1994 JLB : Created.                                                                 *
- *=============================================================================================*/
-inline char const * RawFileClass::File_Name(void) const
```

Comment:
Why does this need to move to the cpp?

Reply:
Just remove the comment block then or pare it down to the minimum? Just trying to keep the diff minimal :D

Reply:
haha. It would have been an option, but for Zero Hour I did the change equal to this. I do not recommend to do this differently now for Generals. Also most other functions also reside in the cpp. There is only one function impl in the header.

Reply:
Okay, you are right, lets try and keep the diff between Generals and Zero Hour down too.